sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/BeamZetaSqlCatalog.java

```diff
@@ -103,6 +107,7 @@
 
   private final Map<List<String>, UserFunctionDefinitions.JavaScalarFunction> javaScalarUdfs =
       new HashMap<>();
+  private final Map<List<String>, Combine.CombineFn<?, ?, ?>> javaUdafs = new HashMap<>();
```

Comment:
I'm suspicious of the `Combine.CombineFn` value. Is this something you were going to change in a followup?

Reply:
Nope, this is how it's going to be. For `AggregateFn`, we will convert it into a `CombineFn` before storing it here.